packages/core/src/base-chart.ts

```diff
@@ -954,4 +1000,30 @@ export class BaseChart {
 	getBBox(selector: any) {
 		return this.innerWrap.select(selector).node().getBBox();
 	}
+// ================================================================================
+	// Makes chart data components more screen reader friendly
+	// ================================================================================
+	makeAccessible (element?: any, dataList?: any) {
+		// Make chart data components tabbable
+		element.attr("tabindex", 0);
+		// Aria label contains the x value, y value, and dataset for the data element
+		if (element.attr("class") === "bar" || element.attr("class") === "dot") {
+			element.attr("aria-label", (d) => `Label: ${d.label}, Value: ${d.value}, belongs to ${d.datasetLabel}`);
```

Comment:
No braces if not necessary.

```suggestion
			element.attr("aria-label", d => `Label: ${d.label}, Value: ${d.value}, belongs to ${d.datasetLabel}`);
```